contracts/common/ConversionHelpers.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.4.24;
+
+
+library ConversionHelpers {
+    function castUintArrayToBytes(uint256[] memory _input) internal pure returns (bytes memory output) {
+        // Force cast the uint256[] into a bytes array, by overwriting its length
+        // Note that the bytes array doesn't need to be initialized as we immediately overwrite it
+        // with the input and a new length. The input becomes invalid from this point forward.
+        uint256 byteLength = _input.length * 32;
+        assembly {
+            output := _input
+            mstore(output, byteLength)
+        }
+    }
+
+    function castBytesToUintArray(bytes memory _input) internal pure returns (uint256[] memory output) {
+        // Force cast the bytes array into a uint256[], by overwriting its length
+        // Note that the uint256[] doesn't need to be initialized as we immediately overwrite it
+        // with the input and a new length. The input becomes invalid from this point forward.
+        uint256 intsLength = _input.length / 32;
+        assembly {
+            output := _input
+            mstore(output, intsLength)
+        }
+    }
+}
```

Comment:
WDYT about having separate helper libraries for uint arrays and bytes? I'm just thinking in case we need to provide more helper functions, having them split by type may make sense.